installation-bundle/src/Resources/views/configuration_error.html.twig

```diff
@@ -27,7 +27,7 @@
         connections:
             default:
                 default_table_options:
-                    engine: MyISAM</pre>
+                    engine: InnoDB</pre>
```

Comment:
@leofeyer Have you seen the message above? 🙂I'm just waiting for a reply if this new approach makes sense for you guys. If it does I would change the pull request according to the approach and this change would be removed anyway. Is this new approach ok for you?

Reply:
Good point. As setting InnoDB in the settings doesn't trigger an error and it's the same without any setting when using the core version, I would just check if `isset($options['engine'])`. If it's not set this error just can't be raised.
The other solution would be to trigger a different error which prompts the user to set `InnoDB` in the config so that it's for sure the same as with the managed edition.
I don't mind ether way, which one would you guys prefer?

Reply:
I don't think we should force advanced users to use a certain DB engine, so I'm in favor of the first solution.

Reply:
Ok, then I will adapt the pull request to this solution.